tests/integration/validate/test_validate_command.py

```diff
@@ -28,8 +28,8 @@ class TestValidate(TestCase):
     @classmethod
     def setUpClass(cls):
         cls.patterns = {
-            TemplateFileTypes.JSON: re.compile(r"template\.json is a valid SAM Template$"),
-            TemplateFileTypes.YAML: re.compile(r"template\.yaml is a valid SAM Template$"),
+            TemplateFileTypes.JSON: re.compile(r"template\.json is a valid SAM Template(\r\n)?$"),
```

Comment:
LGTM! Did we validate that this won't break on Mac or Linux?

Reply:
Ran on Windows and Mac.
The last added part `(\r\n)` is optional `?`